components/style/properties/longhand/inherited_text.mako.rs

```diff
@@ -415,7 +418,7 @@ ${helpers.single_keyword("text-align-last",
         if input.try(|input| input.expect_ident_matching("normal")).is_ok() {
             Ok(SpecifiedValue::Normal)
         } else {
-            specified::LengthOrPercentage::parse_any(input)
+            specified::LengthOrPercentage::parse(_context, input)
```

Comment:
nit: `_context` has the underscore in front because it was an unused variable. Now it's no longer the case so we should rename it to just `context`.